sqlscripts/replication/0007-distributed-city-hash.sql

```diff
@@ -0,0 +1,3 @@
+CREATE TABLE IF NOT EXISTS %s -- global table name
+    ON CLUSTER '{cluster}' AS %s -- local table name
+    ENGINE = Distributed('{cluster}', jaeger, %s, cityHash64(traceID)); -- local table name
```

Comment:
what does the `jaeger` mean? Isn't it a database name?

Reply:
> Isn't it a database name?
Yes, it is.

Reply:
We support database in the config, so it should be automatically adjusted here as well.

Reply:
Database option in config is used for connection. So if we would change this, we won't be able connect to docker clickhouse server out-of-box.

Reply:
> So if we would change this, we won't be able connect to docker clickhouse server out-of-box.

can you explain why? 

My understanding is that if a user sets database in the config all tables will be created in that database.

Reply:
Yes, but if there's no database in server, clickhouse driver won't connect to it.